src/uct/api/v2/uct_v2.h

```diff
@@ -577,6 +596,47 @@ typedef struct uct_iface_is_reachable_params {
 } uct_iface_is_reachable_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_ep_is_connected.
```

Comment:
the is too long for brief, which is usually one sentence. Please split to brief and full